invokeai/frontend/web/src/features/gallery/components/ImageGrid/GalleryImage.tsx

```diff
@@ -121,7 +122,10 @@ const GalleryImage = (props: HoverableImageProps) => {
   }
 
   return (
-    <Box w="full" h="full" className="gallerygrid-image" data-testid={dataTestId}>
+    <Box w="full" h="full" className="gallerygrid-image" data-testid={dataTestId}
+      sx={{
+        containerType: "inline-size",
+      }}>
```

Comment:
This `sx` prop (and the one on the `Text` component) take objects. On every cycle, react will re-create these objects and re-render all children. To fix this we need to give them a stable reference identity. Two ways:
- Define the object outside the component
- Define the object in a useMemo

For this situation, because the `sx` object is static, we can define it outside the component:

```ts
const boxSx: SystemStyleObject = {
  containerType: "inline-size",
}
```

Same goes for the other `sx` prop.